knowrob_map_tools/prolog/semantic_map_utils.pl

```diff
@@ -184,12 +184,15 @@
 % @param H      Object height (z dimension)
 %  
 map_object_dimensions(Obj, W, D, H) :-
-    owl_has(Obj, knowrob:'widthOfObject', literal(type(_, W_))),
-    atom_number(W_, W),
-    owl_has(Obj, knowrob:'depthOfObject', literal(type(_, D_))),
-    atom_number(D_, D),
-    owl_has(Obj, knowrob:'heightOfObject', literal(type(_, H_))),
-    atom_number(H_, H).
+    object_dimensions(Obj, W, D, H).
```

Comment:
Hey, the order in `object_dimensions` is D, W, H, @bbrieber could you please fix this?